src/state/slices/opportunitiesSlice/resolvers/thorchainsavers/index.ts

```diff
@@ -165,64 +164,54 @@ export const thorchainSaversStakingOpportunitiesUserDataResolver = async ({
   }
 
   try {
-    const stakingOpportunityId = accountIdToFeeAssetId(accountId)
-
-    if (!stakingOpportunityId)
-      throw new Error(`Cannot get stakingOpportunityId for accountId: ${accountId}`)
+    for (const stakingOpportunityId of opportunityIds) {
```

Comment:
Pretty much the same implementation as before inside this block, except we're now iterating over all `opportunityIds` each call - which will be the opportunityIds at a given chain i.e AVAX and USDC on Avalanche for Avalanche, ETH, USDC on Ethereum and USDT on Ethereum for Ethereum